rails/Gemfile

```diff
@@ -25,3 +25,7 @@ gem 'sqlite3', '1.3.10'
 gem 'mysql2', '0.3.18'
 gem 'pg', '0.18.1'
 gem 'benchmark-ips', '~> 2.2.0'
+gem 'em-hiredis', '~> 0.3.0'
+gem 'redis', '~> 3.0'
+gem 'faye-websocket'
```

Comment:
I usually look the versions so that we know any performance changes are not due to the gems